tests/pytest/core/test_admin.py

```diff
@@ -71,8 +71,7 @@ def test_pre_login_user_add_staff_to_group(mocker, model_AdminUser):
     mocked_request = mocker.Mock()
     mocked_request.session.get.return_value = None
 
-    mocker.patch.object(benefits.core.admin.settings, "GOOGLE_SSO_STAFF_LIST", [model_AdminUser.email])
-
+    settings.GOOGLE_SSO_STAFF_LIST = [model_AdminUser.email]
```

Comment:
I don't think this is quite right. This is the `settings` object imported from `django.conf`

It should be the `settings` fixture from `pytest-django` (e.g. injected to the test method).

See https://github.com/cal-itp/benefits/blob/main/tests/pytest/test_secrets.py#L80

Reply:
https://github.com/cal-itp/benefits/pull/2347/commits/d35edf76ad79dfa2d958f265da998e1cd7368954